webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-student-button.js

```diff
@@ -21,6 +32,7 @@ class SakaiRubricStudentButton extends RubricsElement {
       entityId: { attribute: "entity-id", type: String },
       toolId: { attribute: "tool-id", type: String },
       evaluatedItemId: { attribute: "evaluated-item-id", type: String },
+      hidden: Boolean,
```

Comment:
Unless you expect this to be an attribute on the element, you should use the full form with attribute: false

```suggestion
      hidden: { attribute: false, type: Boolean },
```